cloud/azure/main.tf

```diff
@@ -60,7 +63,7 @@ resource "azurerm_app_service_plan" "plan" {
 }
 
 resource "azurerm_app_service" "civiform_app" {
-  name                = var.application_name
+  name                = "${var.application_name}-${random_pet.server.id}"
```

Comment:
question(blocking): is this to ensure app and db names are unique? and if so, should it be done in another PR since it doesn't affect the backend config?

Reply:
also random_pet is the best terraform random resource. numbers are so boring!

Reply:
defffinitely could have been in another PR as they were tangentially related (couldn't really test this PR without fixing that, but might be a lil bit of a pain to split up now)

Reply:
I'm fine with leaving it in.